types/ink-spinner/ink-spinner-tests.tsx

```diff
@@ -0,0 +1,10 @@
+/** @jsx h */
+import { h } from 'ink';
+import Spinner from 'ink-spinner';
+// NOTE: `import Spinner = require('ink-spinner');` will work as well.
```

Comment:
Personally, I think it would be better to use the preferred import=require form in the actual code and mention the allowSyntheticDefaultImports variant in the comment.

Reply:
hm, if you look at code examples from TS website I guess none of them use `require` syntax, even in the node app: https://github.com/Microsoft/TypeScript-Node-Starter#typescript-node-starter. so for me it was more sensible to use this variant as something probably more natural for potential users.

Reply:
It's definitely unfortunate that our sample doesn't match our current guidance.  I'm guessing things have evolved as the ES6 module design process has proceeded.  I'll repeat that I, personally, think it's most intuitive to match TS-style exports with TS-style imports but I'm certainly not going to block your contribution on that basis.